content/en/docs/tasks/debug/debug-cluster/kubectl-node-debug.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Debugging Kubernetes nodes with kubectl
+content_type: task
+min-kubernetes-server-version: 1.20
+---
+
+<!-- overview -->
+This page shows how to debug a [node](/docs/concepts/architecture/nodes/) running on the Kubernetes cluster using `kubectl debug` command.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+You need to have access to create Pods and to assign those new Pods to arbitrary nodes.
+You also need to be authorized to create Pods that access filesystems from the host.
+
+
+<!-- steps -->
+
+## Debugging a Node using `kubectl debug node`
+
+Use the `kubectl debug node` command to deploy a Pod to a Node that you want to troubleshoot.
+This command is helpful in scenarios where you can't access your Node by using an SSH connection.
+When the Pod is created, the Pod opens an interactive shell on the Node.
```

Comment:
The only thing possibly missing is stating that `the container on the debugging Pod runs in the host's namespaces`. I'm not going to block on this nit pick.

Reply:
thanks for the reviews @seans3. I would address this nit picks in a follow up PR. 👍🏻